server/src/main/java/org/gluu/oxtrust/service/config/OxAuthConfigurationDAO.java

```diff
@@ -0,0 +1,52 @@
+package org.gluu.oxtrust.service.config;
+
+import org.gluu.oxtrust.api.configuration.OxAuthConfig;
+import org.gluu.oxtrust.config.ConfigurationFactory;
+import org.gluu.persist.PersistenceEntryManager;
+import org.xdi.config.oxtrust.LdapOxAuthConfiguration;
+
+import java.io.IOException;
+
+public class OxAuthConfigurationDAO {
```

Comment:
Let's continue to use old naming convention in this project. Use `Serivce` for reusable methods and `Acrtion` for GUI classes